docker/entrypoint.sh

```diff
@@ -12,79 +12,95 @@ set -eo pipefail
 
 # Exit early if `ZEBRA_CONF_PATH` does not point to a file.
 if [[ ! -f "${ZEBRA_CONF_PATH}" ]]; then
-  echo "the ZEBRA_CONF_PATH env var does not point to a Zebra conf file"
+  echo "ERROR: No Zebra config file found at ZEBRA_CONF_PATH (${ZEBRA_CONF_PATH})."
+  echo "Please ensure the file exists or mount your custom config file and set ZEBRA_CONF_PATH accordingly."
   exit 1
 fi
 
-# Generates a config file for Zebra using env vars set in "docker/.env" and
-# prints the location of the generated config file.
+# Define function to execute commands as the specified user
+exec_as_user() {
+  if [[ "$(id -u)" = '0' ]]; then
+    exec gosu "${USER}" "$@"
+  else
+    exec "$@"
+  fi
+}
+
+# Modifies the existing Zebra config file at ZEBRA_CONF_PATH using environment variables.
 #
-# ## Positional Parameters
+# Environment variables are typically set in "docker/.env".
```

Comment:
The config options this function supports are also listed and documented in the "docker/.env" file.